src/components/Auth/todos/Header/Header.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useEffect, useRef } from 'react';
+
+export const Header: React.FC = () => {
```

Comment:
`Header` is also not a good name here. Try to rewrite checklist, `NewTodo` I think would be better in this case.